lighthouse-core/test/audits/bootup-time-test.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env mocha */
+const BootupTime = require('../../audits/bootup-time.js');
+const fs = require('fs');
+const assert = require('assert');
+
+// sadly require(file) is not working correctly.
+// traceParser parser returns preact trace data the same as JSON.parse
+// fails when require is used
+const acceptableTrace = JSON.parse(
+  fs.readFileSync(__dirname + '/../fixtures/traces/progressive-app-m60.json')
+);
+const errorTrace = JSON.parse(
+  fs.readFileSync(__dirname + '/../fixtures/traces/airhorner_no_fcp.json')
+);
+
+describe('Performance: bootup-time audit', () => {
+  it('should compute the correct BootupTime values', (done) => {
+    const artifacts = {
+      traces: {
+        [BootupTime.DEFAULT_PASS]: acceptableTrace,
+      },
+    };
+
+    const output = BootupTime.audit(artifacts);
+    assert.equal(output.details.items.length, 8);
+    assert.equal(output.score, true);
+    assert.equal(Math.round(output.rawValue), 176);
+
+    const valueOf = name => output.extendedInfo.value[name];
+    assert.deepEqual(valueOf('https://www.google-analytics.com/analytics.js'), {'Evaluate Script': 40.1, 'Compile Script': 9.6, 'Recalculate Style': 0.2});
```

Comment:
It was kind of confusing that our extendedInfo reported tasks, but the UI reported taskGroups. I don't have a strong opinion on what should be exposed in extendedInfo. Maybe we just leave `extendedInfo` empty this time? In the tests we could assert against the table details instead.

Reply:
it might be confusing indeed but isn't it handy to show the raw information somewhere in the json report so people who want to use it can use it? Or isn't that why extendedInfo exists?